core/src/main/java/org/wso2/msf4j/exception/MSF4JOSGiDeclarativeServiceException.java

```diff
@@ -0,0 +1,32 @@
+/*
+*  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.wso2.msf4j.exception;
+
+/**
+ * Runtime exception when MicroServices Service Component fails to add or remove OSGi Declarative Services
+ */
+public class MSF4JOSGiDeclarativeServiceException extends RuntimeException {
```

Comment:
Shall we remove MSF4J prefix from the class name as it has been already included in the package name?